components/slug/SimilarCompanies.tsx

```diff
@@ -0,0 +1,43 @@
+import TableCell from '@mui/material/TableCell';
+import TableRow from '@mui/material/TableRow';
+import CompanyTableData from '../CompanyTableData';
+import styles from '../../styles/Home.module.css';
+import { NoDataTable } from './NoData';
+import { ISimilarCompaniesProps } from '../../types/similarCompanies';
+
+function SimilarCompanies({ data }: ISimilarCompaniesProps) {
+  const companies =
+    data.similar_companies !== null ? (
+      data.similar_companies.map((company) => (
+        <TableRow className={styles.dayRow} key={company.id}>
+          <TableCell>{company?.name}</TableCell>
+          <TableCell>{company?.industry}</TableCell>
+          <TableCell>{company?.location}</TableCell>
+          <TableCell>{company?.site}</TableCell>
+          <TableCell>{company?.email}</TableCell>
+          <TableCell>{company?.phone}</TableCell>
+        </TableRow>
+      ))
+    ) : (
+      <NoDataTable />
+    );
+
+  const row = (
+    <TableRow className={styles.tableRow}>
+      <TableCell className={styles.tableRowTitle}>NAME</TableCell>
+      <TableCell className={styles.tableRowTitle}>INDUSTRY</TableCell>
+      <TableCell className={styles.tableRowTitle}>LOCATION</TableCell>
+      <TableCell className={styles.tableRowTitle}>WEBSITE</TableCell>
+      <TableCell className={styles.tableRowTitle}>EMAIL</TableCell>
```

Comment:
It is best to create a separate modular style file for each component, like:
 ├── 📁  components
        ├── Company
                  ├── Profile
                  ├── WorkHours
                  ├── Header
                  ├── Contacts
                  ├── Info
                            ├── index.tsx
                            ├── index.module.css